provider/resources.go

```diff
@@ -78,8 +78,8 @@ func Provider() tfbridge.ProviderInfo {
 	// Create a Pulumi provider mapping
 	prov := tfbridge.ProviderInfo{
 		P:                    p,
-		Name:                 "tailscale",
-		Description:          "A Pulumi package for creating and managing xyz cloud resources.",
+		Name:                 "Tailscale",
```

Comment:
I believe the provider name must be a lowercase string that matches the string in the plugin's name, like `pulumi-resource-tailscale`.
```suggestion
		Name:                 "tailscale",
```